### ISSUE TITLE
Download Command fails when creating jfrog.cli.temp*

### ISSUE THREAD

**Issue by maiteko** [jfrog/jfrog-cli#824]
My team is having an issue in 1.39.3 where we attempt to sync with a folder on artifactory, the downloads all succeed but the command still fails on creating the temp file. For example:

```
jfrog rt dl --flat=true --threads=8 --sync-deletes="dl_dir" --quiet repo_path dl_dir

 Log path: C:\Users\maiteko\.jfrog\logs\jfrog-cli.2020-09-08.09-29-25.18940.log [---------------]
 Log path: C:\Users\maiteko\.jfrog\logs\jfrog-cli.2020-09-08.09-29-25.18940.log
{
  "status": "failure",
  "totals": {
    "success": 48,
    "failure": 0
  }
}
[Error] mkdir C:\Users\MAITEKO~1\AppData\Local\Temp\jfrog.cli.temp.-1599578966-235641541\C:: The filename, directory name, or volume label syntax is incorrect.
```

it looks like that \C:: is probably the problem.

Because of the failure getting returned, it stops our automated scripts, and prevents other assets from being downloaded.

**Comment by maiteko** [comment]
if it's not obvious, this is specific to windows, I should have said that in the first place.

**Comment by talarian1** [comment]
Hi @maiteko thanks for reporting this issue.
The fix is included in the [1.39.5 release](https://github.com/jfrog/jfrog-cli/releases/tag/1.39.5).
 We would appreciate any feedback from you.

**Comment by maiteko** [comment]
Hey, thanks, that looks like it solved the problem :)
